packages/tomls/src/exchange_pass_nft/configs/mintLotteryPot.toml

```diff
@@ -0,0 +1,9 @@
+[invoke.exchange_pass_lottery_pot_mint_owner]
+target = ["ExchangePassProxy"]
+fromCall.func = "owner"
+func = "mint"
+args = [
+    "<%= settings.lotteryPotExchangeFeeCollectorOwner %>",
+    "<%= solidityKeccak256(['address','uint256'],[settings.lotteryPotExchangeFeeCollectorOwner, 2]) %>",
```

Comment:
what is 2?

Reply:
tbh this is also unclear to me, the reya dex uses hash of address, 1; for the second exchange i did address,2

Reply:
you cannot mint the same token id twice. as long as it's unique it can be minted. It was up to us how we wanted to encode the token id if I remember correctly. the number after the address is just a counter (per address) that allows us to mint multiple nfts for the same address